services/web/server/src/simcore_service_webserver/products/_models.py

```diff
@@ -239,6 +239,8 @@ def _update_json_schema_extra(schema: JsonDict) -> None:
                         "vendor": {
                             "url": "https://acme.com",
                             "license_url": "https://acme.com/license",
+                            "logo": "https://acme.com/logo",
+                            "strong_color": "#123456",
```

Comment:
here I would suggest to add also a ui section instead of flat

Reply:
When these are used in the webserver, we need to also make sure we update products.vendors entries

Reply:
> When these are used in the webserver, we need to also make sure we update products.vendors entries

Do I need to add alarms to the PR title or something?